src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/transcripts-list-item-info/TranscriptsListItemInfo.tsx

```diff
@@ -35,6 +36,9 @@ import { Transcript } from 'src/content/app/entity-viewer/types/transcript';
 
 import transcriptsListStyles from '../DefaultTranscriptsList.scss';
 import styles from './TranscriptsListItemInfo.scss';
+import ViewInApp from 'src/shared/components/view-in-app/ViewInApp';
+import * as urlFor from 'src/shared/helpers/urlHelper';
+import { buildFocusIdForUrl } from 'src/shared/state/ens-object/ensObjectHelpers';
```

Comment:
Could you rearrange the imports please? Move ViewInApp up to InstantDownloadTranscript; move urlFor and buildFocusIdForUrl  up to the rest of the helpers. We usually arrange imports in groups, with libraries from node modules going first, then helpers/services/utils; then components; then types; and the last — styles.